client/apps/edit/components/admin/index.coffee

```diff
@@ -308,17 +308,21 @@ module.exports = class EditAdmin extends Backbone.View
       featuredListTemplate list: @article.featuredList('Artworks')
     )
 
+  renderInputs: (src) ->
+    $('.edit-email-large-image-url input').val(
+      crop(src, { width: 1280, height: 960 } ))
```

Comment:
Do you know if Sailthru does any image processing on their side we could leverage? @dzucconi is helping me look into replacing Embedly and it would be nice to avoid Embedly urls living in email if possible.